### PR TITLE
docs(bigquery/storage/managedwriter): augment docs

### PR DESCRIPTION
**Pull request by shollyman**
Towards: #4366

### CODE REVIEW

**Review by codyoss — APPROVED** [review]
LGTM, nice docs!

File: bigquery/storage/managedwriter/doc.go [hunk 1]
```diff
@@ -24,12 +24,157 @@ feature-rich successor to the classic BigQuery streaming interface, which is pre
...
+	client, err := managedwriter.NewClient(ctx, projectID)
+	if err != nil {
+		// TODO: Handle error.
+	}
+
+
+Defining the Protocol Buffer Schema
+
+The write functionality of BigQuery Storage requires data to be sent using encoded
+protocol buffer messages using proto2 syntax.  As the protocol buffer is not
```
Comment (tswast) [inline bigquery/storage/managedwriter/doc.go:42]: Nit: [Encoding / binary wire format](https://developers.google.com/protocol-buffers/docs/encoding) is the thing the API needs. I think of syntax as the language definition, which doesn't always apply given we can create dynamic messages in some languages.

```suggestion
protocol buffer messages using proto2 wire format.  As the protocol buffer is not
```

**Review by tswast — APPROVED** [review]
Looks like you covered the main topics. Just a minor nit

File: bigquery/storage/managedwriter/doc.go (hunk 1 above)
Comment (shollyman) [reply bigquery/storage/managedwriter/doc.go:42]: Good callout.  I was considering "encoding" before, but was unhappy with it.  Wire format is cleaner.

**Pull request merged**